src/current/v23.1/monitoring-and-alerting.md

```diff
@@ -140,10 +140,12 @@ Otherwise, it returns an HTTP `200 OK` status response code with an empty body:
 ### Raw status endpoints
 
 {{site.data.alerts.callout_info}}
-These endpoints are deprecated in favor of the [Cluster API](#cluster-api).
+The JSON endpoints are deprecated in favor of the [Cluster API](#cluster-api).
+
+The `/_status/vars` metrics endpoint is in Prometheus format and is not deprecated. For more information, refer to [Prometheus endpoint](#prometheus-endpoint).
 {{site.data.alerts.end}}
 
-Several endpoints return raw status metrics in JSON at `http://<host>:<http-port>/#/debug`. Feel free to investigate and use these endpoints, but note that they are subject to change.
+Several endpoints return raw status meta information in JSON at `http://<host>:<http-port>/#/debug`. Feel free to investigate and use these endpoints, but note that they are subject to change.
```

Comment:
```suggestion
Several endpoints return raw status meta information in JSON at `http://<host>:<http-port>/#/debug`. You can investigate and use these endpoints, but note that they are subject to change.
```

Not your change here, I know, but "Feel free" feels kind of informal to me.

Reply:
Same for each version, if you decide to change.

Reply:
good suggestion. updated.